src/reporters/humanReporter.ts

```diff
@@ -18,20 +18,27 @@ import * as os from 'node:os';
 
 export class HumanReporter {
   @elapsedTime()
-  public format(testResult: TestResult, detailedCoverage: boolean): string {
+  public format(
+    testResult: TestResult,
+    detailedCoverage: boolean,
+    concise: boolean = false
+  ): string {
     HeapMonitor.getInstance().checkHeapSize('HumanReporter.format');
     try {
-      let tbResult = this.formatSummary(testResult);
+      let tbResult: string = '';
       if (!testResult.codecoverage || !detailedCoverage) {
-        tbResult += this.formatTestResults(testResult.tests);
+        tbResult += this.formatTestResults(testResult.tests, concise);
       }
 
       if (testResult.codecoverage) {
         if (detailedCoverage) {
-          tbResult += this.formatDetailedCov(testResult);
+          tbResult += this.formatDetailedCov(testResult, concise);
+        }
+        if (!concise) {
+          tbResult += this.formatCodeCov(testResult.codecoverage);
```

Comment:
If concise is set to true, then don't print out the code coverage table at all. However, we still want to pass it to the DetailedCoverage method so we can still print out the failed test methods.